dotnet-client-libraries/YamlDotNet/App/Models/Person.cs

```diff
@@ -0,0 +1,9 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace App.Models;
+
+public class Person
+{
+    [Required] public string Name { get; set; }
```

Comment:
Attributes like this one should be above the property, not on the same line. I know it's a subjective thing, but on here this is the preference.

```suggestion
    [Required]
    public string Name { get; set; }
```